src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,43 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  const active = tabs.find(tab => tab.id === activeTabId) || tabs[0];
```

Comment:
There's a question your variable should answer, who is active?. It's cool when you read a code line like a sentence. 
```suggestion
  const activeTab = tabs.find(tab => tab.id === activeTabId) || tabs[0];
  OR
  const selectedTab = tabs.find(tab => tab.id === activeTabId) || tabs[0];
```

Reply:
> There's a question your variable should answer, who is active?. It's cool when you read a code line like a sentence.

Good point